modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/individual.rb

```diff
@@ -29,12 +29,7 @@ def page4_template_path
         #
         # @param data [Hash] Hash of data to add to the pdf
         def sign_pdf_text(data)
-          @page1_path = if data.dig('representative',
-                                    'type') == 'INDIVIDUAL PROVIDING REPRESENTATION UNDER SECTION 14.630'
-                          insert_text_signatures(page1_template_path, data['text_signatures']['page1'])
```

Comment:
`individual_page1_signatures` in `modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb` should be safe to remove with this change